### PR TITLE
Added Codeup

### PR DESCRIPTION
**Pull request by jotasprout**
Added Codeup info

### CODE REVIEW

File: config/code_schools.yml [hunk 1]
```diff
@@ -438,6 +438,13 @@
   va_accepted: false
   full_time: true
   hardware_included: false
-  
-    
-  
+
+- name: Codeup
+  url: http://codeup.com/
+  address1: 600 Navarro St., #350
```
Comment (rickr) [inline config/code_schools.yml]: Does this show up on the page or is it ignored as a comment? You may have to put this in quotes.

Comment (jotasprout) [inline config/code_schools.yml]: In my terminal, it says the server is working but it won't show anything in my browser so I don't know. It seemed a simple enough change so I wasn't worried. None of the other listings are in quotes--but Codeup should be? The keys and values or just the values? I don't understand (really this time).

Comment (jotasprout) [inline config/code_schools.yml]: If this is something that I can and should test locally, I'll try it on a different computer. Is it not showing up for you?

Comment (jotasprout) [inline config/code_schools.yml]: If this is something that I can and should test locally, I'll try it on a different computer. Is it not showing up for you?

**Comment by jotasprout** [comment]
@rickr If this is something that I can and should test locally, I'll try it on a different computer. Is it not showing up for you?

File: config/code_schools.yml (hunk 1 above)
Comment (jotasprout) [inline config/code_schools.yml]: Ah! Now it's obvious. I'll update and resubmit as soon as I get home. Just copied and pasted the address from their site -- my humblest and most profuse apologies. Also, I was just using "https://localhost:3000" per the documentation. I'll try it with the "/code_schools" later.

**Comment by jotasprout** [comment]
Ah! Now it's obvious. I'll update and resubmit as soon as I get home. Just copied and pasted the address from their site -- my humblest and most profuse apologies. Also, I was just using "https://localhost:3000" per the documentation. I'll try it with the "/code_schools" later.

**Comment by jotasprout** [comment]
Do I need to make another Pull Request? I did one a few hours ago with an updated file that passed the tests but it's not showing up in my activity or here.

**Pull request merged**